django_project_base/account/rest/project_profiles.py

```diff
@@ -141,12 +140,13 @@ def create(self, request: Request, *args, **kwargs) -> Response:
         if "password" in request.data:
             request.data["password"] = make_password(request.data["password"])
         response = super().create(request, *args, **kwargs)
+        user = Profile.objects.get(pk=response.data["id"])
+        try:
+            ProjectMember.objects.create(project=request.selected_project, member=user)
+        except ProjectNotSelectedError:
+            return Response(status=status.HTTP_403_FORBIDDEN)
```

Comment:
lahko tudi raisaš PermissionDenied(e.message) in bo bolj complete response - front eend bo dejansko dobil info, zakaj to ni dovoljeno...